### ISSUE TITLE
installer on latest raspian lite

### ISSUE THREAD

**Issue by tim-moody** [rachelproject/rachelpiOS#30]
Error: Unable to upgrade Raspberry Pi firmware

if I try it manually I get

root@raspberrypi:~# rpi-update
-su: rpi-update: command not found

**Comment by wegunterjr** [comment]
I am getting the Unable to update after running 
`curl -fsS https://raw.githubusercontent.com/rachelproject/rachelpiOS/master/installer.py | python`

I can run rpi-update, but have to do:
`sudo rpi-update`

**Comment by wegunterjr** [comment]
My solution had to do with having a bad source list in 
`/etc/apt/sources.list.d/` 
I removed it, and the update continued.
Also, try 
`sudo apt-get update`
and
`sudo apt-get upgrade`
That is most of what `rpi-update` is doing. 
That might get you an update that contains rpi-update.
It really depends on what version of raspbian you are running. as well.

**Comment by wegunterjr** [comment]
wait, just looked again, why are you root?

**Comment by tim-moody** [comment]
maybe I have habits from fedora.  are you saying I can't update firmware as root?

**Comment by wegunterjr** [comment]
not necessarily.  I think the first question is what version of raspbian are you running?
If it's an older version, try this:
`sudo apt-get install rpi-update`
I think the focus of raspberry pi use is to stay in the default or non-root login, but maybe it's just that your version doesn't have it installed.

**Comment by tim-moody** [comment]
latest - 8.6

**Comment by tim-moody** [comment]
I also use lite rather than full as I like servers to be headless

**Comment by tim-moody** [comment]
I started over with a fresh https://downloads.raspberrypi.org/raspbian_lite_latest (2016-09-23) and this time ran everything as the user pi.
curl -fsS https://raw.githubusercontent.com/rachelproject/rachelpiOS/master/installer.py | python (still as user pi)

I get

Error: Unable to upgrade Raspberry Pi firmware

If I comment out that line in the installer I get to

Error: Unable to set iptables MASQUERADE on eth0

(N.B. I am logged in over ssh to eth0 if that matters.)

**Comment by needlestack** [comment]
Yeah, it looks like Raspbian has changed enough since this release that we need to redo some things. If you want a working RACHEL-Pi build, you might try starting with [one of the pre-built versions here](http://rachelfriends.org/rachel-pi-howto.html) and then modifying from there.

**Comment by needlestack** [comment]
Alternately, if you want to build from the script, you can start with `2016-05-27-raspbian-jessie`. And of course, patches to bring the script up to work with more current versions are welcome :)

**Comment by tim-moody** [comment]
I can pretty much make the current script work, but I have to comment out some lines, which don't seem to be needed anyway.  Just wanted to report the problem.  I don't really want to download a 32G or 64G image to get mostly what I already have downloaded.

**Comment by needlestack** [comment]
Good point. This has come up before -- so I've [added it as an issue for our next release](https://github.com/rachelproject/rachelpiOS/issues/32)

**Comment by tim-moody** [comment]
Is captive portal supposed to work with this installer?  I don't see that it does.  The redirect php and 2 shtml files get installed with the clone of the contentshell, but I don't see any of the iptables rules that are in the rachelplus code except for the nat and standard gateway rules.

**Comment by needlestack** [comment]
No, sorry - we should document that more clearly. The captive portal only
works on the Intel CAP. It would be great if someone were able to port that
functionality to the pi!

**Comment by tim-moody** [comment]
any chance of getting the output of iptables-save on the cap?
